docs/index.rst

```diff
@@ -349,6 +349,22 @@ To set resource flags:
                   }
     >>> result = hs.resource('ID OF RESOURCE GOES HERE').flag(options)
 
+To discover resources via subject or bounding box:
```

Comment:
What about other search/query parameters like date range that is supported by the endpoint? Do we need show an example of using date range?